tools/editor/dependency_editor.cpp

```diff
@@ -420,7 +418,6 @@ void DependencyRemoveDialog::show(const Vector<String> &to_erase) {
 	exist=false;
 	owners->clear();
 	files.clear();
-	TreeItem *root=owners->create_item();
```

Comment:
Note to self: Double check that this change was good, as `TreeItem::create_item()` potentially modifies the `owners` object. (https://github.com/godotengine/godot/blob/dacb620f83c5e84a5f8954bdbbbfa030af9e2444/scene/gui/tree.cpp#L2781)